lib/Basic/LangOptions.cpp

```diff
@@ -72,6 +72,8 @@ static const SupportedConditionalValue SupportedConditionalCompilationOSs[] = {
   "tvOS",
   "watchOS",
   "iOS",
+  "visionOS",
+  "xrOS",
```

Comment:
I forget - did we implement migration warnings from the toolchain name to the platform name?

Reply:
I don't think we have, but we really ought to. I will take a look at that once this change lands.